### DISCUSSION TITLE
Help with initial configuration

### DISCUSSION THREAD

**Discussion by deadbeatdandylyon** [FlareSolverr/FlareSolverr#800] [Q&A]
Hello, I am new to this program and I want to see if it will help with what I'm trying to do however I do not know the first thing about python I did download the exe download and unzipped it but I really don't know what to do after that point. would someone be willing to walk me through on getting started. 

Thank you

**Comment by ilike2burnthing** [comment]
See the README.

What are you trying to use it for?

**Comment by deadbeatdandylyon** [comment]
I run a dmca service, and there are a bunch of sites that are hidden behind cloudflare and i'm looking to find the original ip address that is hidden behind the cloudflare. i did a quick google search of how to find an ip address behind cloudflare and i first found this but it is no longer active - https://github.com/zidansec/CloudPeler and so i searched alternatives to this service and found this one so it might not be what i'm looking for in which case i'll move on but if it or can help then yes would need some help as i have no idea on how to set it up

**Reply by ilike2burnthing** [reply]
No, FlareSolverr doesn't reveal public IPs.
